draft-ietf-capport-architecture.xml

```diff
@@ -347,13 +352,13 @@
         <t>
         User Equipment may send traffic outside the captive network prior to the Enforcement
         device granting it access. The Enforcement Device rightly blocks or resets these
-        requests. However, lacking a signal from the Enforcement Device or interaction with the
+        requests. However, in the absence of a signal from the Captive Network or interaction with the
```

Comment:
Is there a particular reason you preferred referring to the Captive Network as opposed to the Enforcement Device? The Enforcement Device is the component within the network responsible for sending the signal. It makes sense to me that it be the thing that has not sent the signal to the User Equipment in this paragraph.

Reply:
You're right, of course. I made this edit without reading the entire section (when I was slightly confused, as I noted in my email), and forgot to remove this edit.

509d855e5ddba7dd1c5db58646c85516d0b40987 corrects it.